src/api/.rubocop_todo.yml

```diff
@@ -169,6 +169,7 @@ Lint/NonLocalExitFromIterator:
     - 'app/models/bs_request.rb'
     - 'app/models/patchinfo.rb'
     - 'app/models/relationship.rb'
+    - 'app/controllers/webui/package_controller.rb'
```

Comment:
Whats this about?

Reply:
rake dev:lint is complaining about it:

```
Offenses:

app/controllers/webui/package_controller.rb:123:7: W: Lint/NonLocalExitFromIterator: Non-local exit from iterator, without return value. next, break, Array#find, Array#any?, etc. is preferred.
      return
      ^^^^^^
```
even if I didn't touch it https://github.com/vpereira/open-build-service/blob/verify_authorized_on_packages/src/api/app/controllers/webui/package_controller.rb#L123, To fix it, looks like the whole ```dependency``` action should be rewritten .. what do you think @hennevogel ?